app/concepts/tickets/use_cases/update.rb

```diff
@@ -0,0 +1,15 @@
+module Tickets
+  module UseCases
+    class Update
+      attr_reader :repository
```

Comment:
And here is an example, you didn't declare :params and :id here. I guess it does work anyway, doesn't it?